web-api/src/mind.rs

```diff
@@ -187,6 +194,12 @@ impl Default for PersonaBasedConfig {
     }
 }
 
+#[derive(Debug)]
+struct TestConfig {
+    coi_config: CoiConfig,
+    personalization_config: PersonalizationConfig,
```

Comment:
:art: i'd rather call it `StateConfig` as it is used as the config for the `State`, wdyt?

:art: we can also make the field names less redundant, they are already part of a config type.

```suggestion
struct StateConfig {
    coi: CoiConfig,
    personalization: PersonalizationConfig,
```

Reply:
agree